pulldown-cmark/src/lib.rs

```diff
@@ -725,8 +725,10 @@ bitflags::bitflags! {
         /// ```markdown
         /// title 1
         ///   : definition 1
+        ///
```

Comment:
I can confirm that the blank line is also [required by pandoc](https://pandoc.org/try/?params=%7B%22text%22%3A%22title+1%5Cn++%3A+definition+1%5Cntitle+2%5Cn++%3A+definition+2%22%2C%22to%22%3A%22html5%22%2C%22from%22%3A%22commonmark_x%22%2C%22standalone%22%3Afalse%2C%22embed-resources%22%3Afalse%2C%22table-of-contents%22%3Afalse%2C%22number-sections%22%3Afalse%2C%22citeproc%22%3Afalse%2C%22html-math-method%22%3A%22plain%22%2C%22wrap%22%3A%22auto%22%2C%22highlight-style%22%3Anull%2C%22files%22%3A%7B%7D%2C%22template%22%3Anull%7D), so this really was a bug in the docs, not a bug in the implementation.